LICENSE

```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2016, Brian McFee
+## ISC License
```

Comment:
I'm not sure we can have / want a blanket license on this repo.  Each example might be licensed individually, and I'd like to allow some wiggle room there.  Similarly, as you noted below, the example tracks have different licenses from the code.